dashboard/src/components/dashboard/GuildCard.vue

```diff
@@ -70,7 +70,6 @@ export default {
 	background-position: -99.99% 0;
 }
 .guild-logo {
-	height: 10em;
-	width: 10em;
+	height: 5em;
```

Comment:
Images keep their aspect ratio by default. Only define both height and width if you want to override that.